packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/dto/HideDiagramElementInput.java

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.collaborative.diagrams.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.text.MessageFormat;
+import java.util.Objects;
+import java.util.Set;
+import java.util.UUID;
+
+import org.eclipse.sirius.components.collaborative.diagrams.api.IDiagramInput;
+
+/**
+ * The class of the inputs for the "hide element" mutation.
+ *
+ * @author tgiraudet
+ */
+public class HideDiagramElementInput implements IDiagramInput {
+
+    private UUID id;
+
+    private String editingContextId;
+
+    private String representationId;
+
+    private Set<String> elementIds;
+
+    @JsonProperty("hide")
```

Comment:
Can be removed

Reply:
I wondered why you needed to use that but I found out. Since you are writing way too much code manually instead of relying of the IDE, your code does not follow the Java recommend practices. As a result, the field hide is not properly detected since the accessor is named `hideElement`.

You should use your IDE more and type less code manually.